### ISSUE TITLE
Testing instructions

### ISSUE THREAD

**Issue by mentionbottest** [facebook/mention-bot#17]
I made an account for my bot, and was able to get it to comment on the example PR, but I'm not clear on how I can use my version of the bot on one of my repos to test if my changes are working correctly. Could more detailed instructions be added on how to test?

**Comment by vjeux** [comment]
You need to host your bot somewhere (heroku is pretty easy to get started with) and add the url where it is hosted at as a webhook.

If you get it working and want to submit a pull request with the instructions that would be great

**Comment by jgardella** [comment]
Just realized I accidentally commented with my bot's account. Whoops.

I was able to get the bot set up on heroku, and it's commenting on my test repo, but it always comments with "@none". This is using the original source of the bot, with none of my changes. I figure there must just be some step I'm missing. Any ideas, @vjeux?

**Comment by vjeux** [comment]
I would put console.log in all the places and try to figure out what's going on.

**Comment by jgardella** [comment]
I looked into this a bit more, and I discovered the blame was not being parsed correctly. For some reason, the HTML for my test repo does not match the form that is expected. The regex in the parseBlame function is searching for `rel="contributor"`, as seen in [the HTML for this blame](https://github.com/facebook/mention-bot/blame/master/README.md), but if you inspect the [HTML on the blame of my test repo](https://github.com/mentionbottest/test/blame/master/README.md), you'll notice it instead has `rel="author"`.

I'm not sure what is causing this discrepancy.
